core/templates/dev/head/pages/exploration_editor/EditorNavigationDirective.js

```diff
@@ -94,10 +101,15 @@ oppia.directive('editorNavigation', [
               windowClass: 'oppia-help-modal'
             });
 
-            modalInstance.result.then(function() {
-              $rootScope.$broadcast('openEditorTutorial');
+            modalInstance.result.then(function(mode) {
+              if (mode === 'editor') {
+                $rootScope.$broadcast('openEditorTutorial');
+              } else if (mode === 'translation') {
+                $rootScope.$broadcast('openTranslationTutorial');
+              }
             }, function() {
-              StateEditorTutorialFirstTimeService.markTutorialFinished();
+              StateTutorialFirstTimeService.markEditorTutorialFinished();
+              StateTutorialFirstTimeService.markTranslationTutorialFinished();
```

Comment:
Do we have to mark both of the tutorials to be finished at the same time?

Reply:
Can you explain the reasoning why the tutorial needs to be marked as finished when the user opens the help modal but cancels ? I have been wanting to ask about this but I keep forgetting.

Reply:
I'm too not sure, can you please go through this and fix it?

Reply:
Removed this, could not find a good reason for keeping even after looking at file history and testing for usefulness.